tests/test_solver_based_model.py

```diff
@@ -187,7 +187,7 @@ def test_set_bounds_scenario_1(self, core_model):
         assert acald_reaction.forward_variable.ub == 1000.
         assert acald_reaction.reverse_variable.lb == 0
         assert acald_reaction.reverse_variable.ub == 1000.
-        acald_reaction.upper_bound = acald_reaction.lower_bound - 100
+        acald_reaction.bounds = (acald_reaction.lower_bound - 100,) * 2
```

Comment:
I have the suspicion that writing out the full tuple instead of `* 2` is easier to grok. I know it creates a pair but it might be misunderstood since conceptually we're dealing with numerical values here.